plan_delegator/CMakeLists.txt

```diff
@@ -1,9 +1,23 @@
+# Copyright (C) 2018-2019 LEIDOS.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
 cmake_minimum_required(VERSION 2.8.3)
 project(plan_delegator)
 
 ## Compile as C++11, supported in ROS Kinetic and newer
 add_compile_options(-std=c++11)
-set( CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -Wall")
```

Comment:
Interesting that you need -pthread here, does ROS not automatically compile with pthread support? Could you describe a bit the high-level threading structure if you have one in here?

Reply:
Ops. It is something I used to use but I do not think we still need it to be here.